addons/test_mail/tests/test_mail_gateway.py

```diff
@@ -1535,3 +1537,49 @@ def test_message_cc_update_no_old(self):
                                 cc='cc2 <cc2@example.com>, cc3@example.com', target_model='mail.test.cc')
         cc = email_split_and_format(record.email_cc)
         self.assertEqual(sorted(cc), ['"cc2" <cc2@example.com>', 'cc3@example.com'], 'new cc should have been added on record (unique)')
+
+
+@tagged('mail_gateway')
+class TestMailThread(MailCommon):
```

Comment:
There are already a bunch of tests for `message_route` in `TestMailGateway`, no need to add a whole new test class for it ^^